### PR TITLE
[Add] circles homework

### PR DESCRIPTION
**Pull request by petardmnv**

### CODE REVIEW

File: homework_03/17_petar_damyanov/test_circle.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from circle import Circle, Point, RelativePosition
+
+def test_no_common_points():
```
Comment (Vic-Dim) [inline homework_03/17_petar_damyanov/test_circle.py:3]: It's good you've done multiple tests for a single case, it ensures the code works with various parameters

File: homework_03/17_petar_damyanov/circle.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    def __init__(self, center, r):
+        assert isinstance(center, Point) and isinstance(r, float)
+        self.center = center
+        self.r = r
+
+    def find_relative_position(self, second_circle):
+        assert isinstance(second_circle, Circle)
+
+
+        centers_distance = sqrt( pow((self.center.x - second_circle.center.x), 2) + pow((self.center.y - second_circle.center.y), 2) )
```
Comment (Vic-Dim) [inline homework_03/17_petar_damyanov/circle.py:30]: The variables you're using for accessing circle's attributes are simple enough

File: homework_03/17_petar_damyanov/circle.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+        if (self.center.x == second_circle.center.x and self.center.y == second_circle.center.y and self.r == second_circle.r):
```
Comment (Vic-Dim) [inline homework_03/17_petar_damyanov/circle.py:33]: Some people tend to not use if-elif clause, which is actually faster than doing multiple if clauses. Good one

**Review by Vic-Dim — APPROVED** [review]

File: homework_03/17_petar_damyanov/circle.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        self.x = x
+        self.y = y
+
+
+class Circle:
+
+    def __init__(self, center, r):
+        assert isinstance(center, Point) and isinstance(r, float)
+        self.center = center
+        self.r = r
```
Comment (Lilly7777) [inline homework_03/17_petar_damyanov/circle.py:24]: You can add a condition r > 0.

File: homework_03/17_petar_damyanov/test_circle.py (hunk 1 above)
Comment (Lilly7777) [inline homework_03/17_petar_damyanov/test_circle.py:3]: The tests cover the whole method.

**Review by Lilly7777 — APPROVED** [review]
Bravo Pesho!

**Pull request merged**